test/implicit_func.c

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+int main()
+{
```

Comment:
Can you put this in `test/other`, and put opening curly the same line as the function decl.